bc/assets/templates/base.html

```diff
@@ -1,4 +1,4 @@
-{% load static tailwind_tags %}
+{% load static %}{% load tailwind_tags %}
```

Comment:
I'm pretty sure you can do this in newer versions of Django:

```suggestion
{% load static tailwind_tags %}
```

Reply:
You're right. We can do that with the `load` tag